Added local telemetry file and survey details. (#8654)
flutter/website: src/reference/user-surveys.md

---
title: User surveys
description: Why users see a survey announcement, how the data is used, and how to disable.
---

## Why do I see a survey announcement?
If you have not opted-out of Flutter’s 
[analytics and crash reporting]({{site.url}}/reference/crash-reporting), 
you may receive a survey announcement in your IDE. 

We run two types of surveys:
1. Quarterly surveys - For these surveys, we notify all active
 Flutter and Dart users to optionally take the survey.
2. Ad-hoc surveys (experimental): We design and deploy these 
surveys when we want to learn more about specific topics. 
You may see a survey announcement if you meet the criteria of 
the survey  based on data stored in the 

> `dart-flutter-telemetry.log` file on your disk.

Reviewer: Same thing here for where they can find this file in the HOME directory on macOS/linux and on windows it will be in the `AppData` directory

Author: Thanks!! Does the following description make sense? 

Additionally, you may review the recently reported data in the
`.dart-tool/dart-flutter-telemetry.log` file under the home directory
`~/` (macOS/Linux) or the `<username>\AppData` (Windows) directory.

Reviewer: Perfectly put!

---
title: User surveys
description: Why users see a survey announcement, how the data is used, and how to disable.
---

## Why do I see a survey announcement?
If you have not opted-out of Flutter’s 
[analytics and crash reporting]({{site.url}}/reference/crash-reporting), 
you may receive a survey announcement in your IDE. 

We run two types of surveys:

> 1. Quarterly surveys - For these surveys, we notify all active
>  Flutter and Dart users to optionally take the survey.
> 2. Ad-hoc surveys (experimental): We design and deploy these 
> surveys when we want to learn more about specific topics. 
> You may see a survey announcement if you meet the criteria of 
> the survey  based on data stored in the 
> `.dart-tool/dart-flutter-telemetry.log` file under the home directory
> `~/` (macOS/Linux) or the `<username>\AppData` (Windows) directory. 

Reviewer 2: **issue:** Use parallel structure, remove the parenthetical, and tighten the language. Remove location of the file. That's extra info that doesn't aid in understanding here.

```suggestion
1. **Each quarter.** We give all active Flutter and Dart users
  the option to take this survey.
2. **Ad-hoc.** We design and deploy experimental surveys
  when we want to learn more about specific topics. 
  If your telemetry data meets the survey criteria,
  you might see a survey announcement. 
```

## How will my responses be used?

> Your responses submitted via survey will be used to improve 
> Flutter and Dart. This information will be stored independently 
> from the information sent to Google via analytics. You can see 
> how previous surveys were used to improve Flutter and Dart on 
> [Medium][]. 

Reviewer 2: **issue:** Write in active voice per [GSG](https://developers.google.com/style/voice). 
Condition, then instruction per [GSG](https://developers.google.com/style/sentence-structure).

```suggestion
We use the responses you submit via survey to improve 
Flutter and Dart. We store this information independent 
of the information sent to Google via analytics.
To see how we used prior surveys to improve Flutter and Dart,
check out our blogs on [Medium][]. 
```

## How can I disable it?

> You may mute survey announcements by clicking a button on the 
> message. Or you may choose to opt-out of analytics and crash 
> reporting, by following the steps described in the “Disabling 
> analytics reporting” section of the [Flutter crash reporting page]({{site.url}}/reference/crash-reporting).

Reviewer 2: **issue:** 
Condition, then instruction per [GSG](https://developers.google.com/style/sentence-structure).

```suggestion
To mute survey announcements, you might do one of the following:

1. Click the button on the message.
2. Opt-out of analytics and crash reporting per the steps
  given in
  [Disabling 
analytics reporting]({{site.url}}/reference/crash-reporting#disabling-analytics-reporting).
```
